03_ocp_repo_sync.sh

```diff
@@ -21,7 +28,12 @@ function sync_go_repo_and_patch {
     git checkout master
     git pull --rebase origin master
     git branch -D we_dont_need_no_stinkin_patches || true
-    git checkout -b we_dont_need_no_stinkin_patches
+    if [[ -n "$pr" ]]; then
+        git fetch origin "pull/$pr/head:we_dont_need_no_stinkin_patches"
```

Comment:
Sometimes PRs get stale, to ensure we always test the latest state (especially in CI) should we perhaps fetch the branch then rebase on master?

Reply:
+1 to rebasing, things are still changing quickly.